test/performance/odp_crypto.c

```diff
@@ -32,17 +32,26 @@
 
 static uint8_t test_iv[8] = "01234567";
 
-static uint8_t test_key16[16] = { 0x01, 0x02, 0x03, 0x04, 0x05,
+static uint8_t test_key24[24] = { 0x01, 0x02, 0x03, 0x04, 0x05,
```

Comment:
`non tangere testos meos`

Reply:
not sure what do u mean here.

Reply:
"Do not change tests". See my previous comment.

Reply:
I have updated the tests so that it works with both the platforms linux-generci & linux-dpdk.

Reply:
Could you please add tests cases rather than modify existing ones?